style.css

```diff
@@ -224,8 +224,6 @@ ul {
   gap: 8px;
   margin: 12px 0;
   justify-content: flex-start;
-
-  /* display: flex; */
 }
 
```

Comment:
- [ ] Kindly make sure your whole design is responsive, by ensuring that it does not break as long as we are between 375 and 1440px of width
![rawcdn githack com_adeola003_Microverse_portfolio_cf8ba025af8d09916fc1c705bd0b1760652d4a3d_index html](https://user-images.githubusercontent.com/97834160/220878684-da4a04bc-06f4-4505-852a-2c5ce32bdd03.png)